docs/dev/table/tableApi.md

```diff
@@ -1682,6 +1682,36 @@ The `OverWindow` defines a range of rows over which aggregates are computed. `Ov
 
 {% top %}
 
+### Aggregate
+
+Aggregate performs an aggregate operation with an aggregate function. You have to close the
+"aggregate" with a select statement. The output will be flattened if the output type is a
```

Comment:
Add a sentence that can't use an aggregate function in the select statement as it has been checked in the select.

Reply:
Done